nixos/modules/services/web-apps/hotcrp.nix

```diff
@@ -0,0 +1,199 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
```

Comment:
```suggestion
```
to address #208242 for this module, uses of `with` such as this should be avoided.
consider `inherits` where you find yourself using `lib.foo` excessively.
[this](https://nix.dev/guides/best-practices.html#with-scopes)  nix.dev page on best practices with `with` may be helpful